src/main/java/redis/clients/jedis/authentication/TokenCredentials.java

```diff
@@ -0,0 +1,24 @@
+package redis.clients.jedis.authentication;
+
+import redis.clients.authentication.core.Token;
+import redis.clients.jedis.RedisCredentials;
+
+class TokenCredentials implements RedisCredentials {
+    private final String user;
+    private final char[] password;
+
+    public TokenCredentials(Token token) {
+        user = token.tryGet("oid");
```

Comment:
Should this be moved to the core library?
e.g. 
Token.getAuthority() to abstract in case if any future supported IdentyProvider provides the username in different attribute?

Reply:
i am on it